### ISSUE TITLE
代码中的一些可能bug?

### ISSUE THREAD

**Issue by long2double** [dingwoai/FairMOT-BDD100K#15]
两个问题

第一个问题：
multitracker.py中update_sep3函数中已用F.normalize,
`
with torch.no_grad():
    output = self.model(im_blob)[-1]
    hm = output['hm'].sigmoid_()
    wh = output['wh']
    id_feature = output['id']
    id_feature = F.normalize(id_feature, dim=1)
`
然而，STrack类中，update_features方法中，再次使用np.linalg.norm(feat)是否已经重复归一化操作？（需要去掉F.normalize???）

第二个问题：
update_sep3函数中，504行
for i_class in range(1, self.opt.num_classes + 1):
            activated_stracks = []
            refind_stracks = []
            lost_stracks = []
            removed_stracks = []
            dets = dets_classes[i_class]
            if conf_thres is None:
                remain_inds = dets[:, 4] > self.opt.conf_thres
            else:
                remain_inds = dets[:, 4] > conf_thres[i_class-1]
            dets = dets[remain_inds]
            ############是否少了索引remain_inds?????###############
            id_feature = cls_id_feats[i_class-1]

**Comment by dingwoai** [comment]
Indeed!!!
Thx a lot!

For the second one, did you get better results after fixing this?

**Comment by long2double** [comment]
暂时还没有进行训练，只是看了下源码，后续有结果在联系
